types/forge-apis/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Forge-apis 0.4
+// Type definitions for Forge-apis 0.7
 // Project: https://github.com/Autodesk-Forge/forge-api-nodejs-client
 // Definitions by: Autodesk Forge Partner Development <https://github.com/Autodesk-Forge>
```

Comment:
You should add your name and account link on a new line here

Reply:
done

Reply:
Sorry, check the format against another package:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ebc4c80b7dc824d0e9d864b076dde5a93bae07a2/types/react/index.d.ts#L3-L24

I’m not sure whether our infrastructure understands it with a comma separator on a single line, but I’m guessing not.

Reply:
Yes comma separated accounts can be understood as per [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/forge-viewer/index.d.ts) - you can see Alan is being recognized as an owner.